plugins/connectrpc/python/v0.4.2/buf.plugin.yaml

```diff
@@ -0,0 +1,17 @@
+version: v1
+name: buf.build/connectrpc/python
+plugin_version: v0.4.2
+source_url: https://github.com/connectrpc/connect-python
+description: Generates client and server stubs for Connect Python. Compatible with the Connect RPC protocol.
```

Comment:
no integration_url yet, because we [haven't moved the docs over to connectrpc.com](https://github.com/connectrpc/connect-python/issues/16).